test/autorest/generated/stringgroup/internal/stringgroup/enum.go

```diff
@@ -115,4 +115,3 @@ func (EnumOperations) PutReferencedConstantHandleResponse(resp *azcore.Response)
 	}
 	return &EnumPutReferencedConstantResponse{StatusCode: resp.StatusCode}, nil
 }
```

Comment:
The linter is going to complain about this isnt it? (The missing endline)

Reply:
There were two `\n` characters at the end, the extra one was removed.

Reply:
Do we want to leave it like that?

Reply:
It's just an artifact of the codegen.  While we could fix it in the code generator to not emit the extra `\n', since we're going to run `gofmt` anyways I think it's easier to let the formatting tool fix it.

Reply:
Ok sounds good 👍